anjumol/react_new/section_11/src/App.jsx

```diff
@@ -0,0 +1,108 @@
+import { useRef, useState, useEffect, useCallback } from 'react';
+
+import Places from './components/Places.jsx';
+import { AVAILABLE_PLACES } from './data.js';
+import Modal from './components/Modal.jsx';
+import DeleteConfirmation from './components/DeleteConfirmation.jsx';
+import logoImg from './assets/logo.png';
+import { sortPlacesByDistance } from './loc.js';
+
+const storedIds = JSON.parse(localStorage.getItem('selectedPlaces')) || [];
+const storedPlaces = storedIds.map((id) =>
+  AVAILABLE_PLACES.find((place) => place.id === id)
+);
+
+function App() {
+  const selectedPlace = useRef();
```

Comment:
change the variable name selectedPlace to selectedPlaceRef also change other places in this file